parm/convection_permitting/namelist.atmosphere

```diff
@@ -33,7 +33,8 @@
     config_block_decomp_file_prefix  = 'graphinfo/${MESH_NAME}.graph.info.part.'
 /
 &restart
-    config_do_restart = ${do_restart}
+    config_do_restart = false
+    config_do_DAcycling = ${do_DAcycling}
```

Comment:
Can `config_do_DAcycling` handle `init.nc` and `mpasout.nc` files automatically?

Reply:
Yes, after adding a set of surface fields into mpasout.

Reply:
Thanks, @hu5970
At which step do we add those surface fields into mpasout?

Reply:
Thanks for more information. Since we use GSL physics, has GSL's version of MPAS added those surface fields into the mpasout files?